src/Compilers/Core/Portable/CodeGen/ItemTokenMap.cs

```diff
@@ -71,7 +68,7 @@ public IEnumerable<string> GetAllItems()
 
         //TODO: why is this is called twice during emit?
         //      should probably return ROA instead of IE and cache that in Module. (and no need to return count)
-        public IEnumerable<string> GetAllItemsAndCount(out int count)
+        public IEnumerable<T> GetAllItemsAndCount(out int count)
```

Comment:
In that case, feel free to leave the signature of `GetAllItems()` as is. Changing it to return an `ImmutableArray<T>` will mean we'll end up boxing the struct when using it as `IEnumerable<T>` from the CCI consumer.